3p/recaptcha.js

```diff
@@ -165,12 +165,18 @@ function actionTypeHandler(win, grecaptcha, data) {
           );
         },
         function(err) {
-          user().error(TAG, '%s', err.message);
+          let message =
```

Comment:
Ha! Interesting, Didn't know the error can be null. In such case is it still a valid error that we want to report. Or can we ignore them?

Reply:
So the output of `err` is undefined, so I do think it is worth reporting. Because usually I have seen this in bugs as a configuration error with the user's recaptcha dashboard, and not an error on us 😄

Reply:
SG